extension.js

```diff
@@ -144,6 +160,9 @@ function on_activate({ give_feedback }) {
     if (give_feedback) {
       show_osd(null, false, microphone.level);
     }
+    if(settings.get_boolean("play-feedback-sounds")) {
```

Comment:
A nitpicking, but this project seems to use `if<space>(` formatting convention, while you seem to prefer to omit a space character (multiple times). Maybe it would be good to keep the original style?